api/models/v0.2/modelDebug.js

```diff
@@ -0,0 +1,38 @@
+const filter = require('underscore').filter;
+const isEmpty = require('underscore').isEmpty;
```

Comment:
Does the `import { filter, isEmpty } from 'underscore';` syntax work here? Just putting out another option for syntax but not a big deal.

Reply:
If I overwrite lines 1 and 2 with `import { filter, isEmpty } from 'underscore';` I see `SyntaxError: Unexpected token import` when running the tests.

Apparently this isn't supported by default yet: https://stackoverflow.com/questions/39436322/node-js-syntaxerror-unexpected-token-import